### ISSUE TITLE
 google play console rejected due to java error

### ISSUE THREAD

**Issue by shaikhkhamroddin** [i18next/react-i18next#1736]
dev, staging and internal testing working properly, but getting rejected from google play console due to below error :

Exception java.lang.RuntimeException: java.lang.IllegalStateException: Source and target locales are the same
at android.app.job.JobServiceEngine$JobHandler.handleMessage (JobServiceEngine.java:118)
at android.os.Handler.dispatchMessage (Handler.java:106)
at android.os.Looper.loopOnce (Looper.java:201)
at android.os.Looper.loop (Looper.java:288)
at android.app.ActivityThread.main (ActivityThread.java:7918)
  at java.lang.reflect.Method.invoke
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:548)
at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:936)
Caused by java.lang.IllegalStateException: Source and target locales are the same
at mnb.bw
  at exp.a (PG:9)
  at exq.b
at com.google.[android.app](http://android.app/)s.miphone.aiai.translate.settings.jobservice.AutoTranslateSettingJobService.onStopJob (PG:5)
at [android.app](http://android.app/).job.JobService$1.onStopJob (JobService.java:73)
at [android.app](http://android.app/).job.JobServiceEngine$JobHandler.handleMessage ([JobServiceEngine.java:114](http://jobserviceengine.java:114/))

Model - Pixel 7 Android 13 (SDK 33)
Release version - 13
SDK version - 33
ABI - arm64-v8a
OpenGL ES - 3.2

Here are some app build settings:
Min API Level - Android 10
Target API Level - Android 13 (API 33)
Arm

**Comment by adrai** [comment]
Sorry, I have no idea what you mean...
react-i18next is a JavaScript project, not a Java...
I don't think this is related to react-i18next at all...
Just make sure you add an intl polyfill if necessary: https://www.i18next.com/translation-function/plurals

**Comment by shaikhkhamroddin** [comment]
yes @adrai I know it's javascript project, but once we create build, code gets converted to java files. Everything is working fine upto internal testing but in when app sent for review in GPC then it gets rejected due to above issue. Not getting any error and all in dev mode.

**Comment by adrai** [comment]
And why do you think that error is related to react-i18next?

**Comment by shaikhkhamroddin** [comment]
as the error is related to locales.

**Comment by adrai** [comment]
Sorry, but I don't think this is at all related to react-i18next...
I'm not able to help, sorry.

**Comment by jamuhl** [comment]
99.99% not looking like related to react-i18next -> the error message seems more related to some call to some translation service needing source and target locale...

https://forum.unity.com/threads/google-play-console-error-java.1510811/ 

even states more details:

```js
  at mnb.bw
  at exp.a (PG:9)
  at exq.b
  at com.google.android.apps.miphone.aiai.translate.settings.jobservice.AutoTranslateSettingJobService.onStopJob (PG:5)
  at android.app.job.JobService$1.onStopJob (JobService.java:73)
  at android.app.job.JobServiceEngine$JobHandler.handleMessage (JobServiceEngine.java:114)
```

So it's reasonable to search what causes the error on `AutoTranslateSettingJobService`

**Comment by CuiGuoYin** [comment]
I suggest repackaging and trying it out again

**Comment by adrian-bowtie** [comment]
@shaikhkhamroddin im having the exact same error. did you find out what was the issue? thanks

**Comment by shaikhkhamroddin** [comment]
> @shaikhkhamroddin im having the exact same error. did you find out what was the issue? thanks

@adrian-bowtie , app got successfully published. This issue is android internal OS issue, it's not related to our app development.
